src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -370,6 +373,12 @@
   - name: --max-count
     type: int
     short-summary: Maximum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specify the value in the range of [1, 100]
+  - name: --clear-cluster-autoscaler-profile
+    type: bool
+    short-summary: Clears the cluster-autoscaler profile.
+  - name: --cluster-autoscaler-profile
+    type: list
+    short-summary: Space-separated list of key=value pairs for configuring cluster autoscaler.
```

Comment:
Maybe you use empty string to represent clear operation. Then we can delete --clear-cluster-autoscaler-profile and only reserve --cluster-autoscaler-profile. For example, `--cluster-autoscaler-profile ""`

Reply:
If we go with this, with that example a user does:
```
az aks update {{some stuff}}
az aks update --cluster-autoscaler-profile ""
```
I could be missing something here but how do we differentiate between both intentions in the validator? To the validator in both cases, `namespace.cluster_autoscaler_profile` is falsy but the intention in the first command is to not clear the profile while the intention of the second one is to clear it.

Reply:
This is the only request of mine. After this is revolved, I'll approve.